akri/Setup-Akri.ps1

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+<#
+    .PARAMETER installAkri
+        installAkri boolean flag
+    
+    .PARAMETER uninstallAkri
+        uninstallAkri boolean flag
+
+    .PARAMETER kubeConfigFile
+        clusterName Path
+
+    .PARAMETER namespace
+        namespace
+#>
+param (
+    [Parameter()]
+    [bool] $installAkri,
+
+    [Parameter()]
+    [bool] $uninstallAkri,
+
+    [Parameter(Mandatory = $true)]
+    [string] $kubeconfigFile,
+        
+    [Parameter(Mandatory = $true)]
+    [string] $namespace
+)
+
+function Install-Akri {
+    <#
+    .DESCRIPTION
+        Helper function to install the Akri addon on the target cluster.
+
+    .PARAMETER kubeConfigFile
+        cluster kubeConfigFile
+
+    .PARAMETER namespace
+        namespace
+    #>
+
+    param (
+        [Parameter(Mandatory = $true)]
+        [String] $kubeConfigFile,
+        
+        [Parameter(Mandatory = $true)]
+        [String] $namespace
+    )
+
+    Write-Host "Installing Akri"
+    Write-Host "Creating namespace $namespace"
+    $namespacecheck = kubectl.exe --kubeconfig=$kubeConfigFile get namespace | findstr $namespace

```

Comment:
Should we do a check to ensure that kubectl with this kubeconfig is connecting ok to the cluster?